apps/example-orchestrator-ui/components/StartPage/ListsRowStartPage.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { ReactElement, useState } from 'react';
+import { EuiFlexGroup } from '@elastic/eui';
+import { useQuery } from 'react-query';
+import { ORCHESTRATOR_API_BASE_URL } from '../../constants';
+import ListStartPage from './ListStartPage';
+
+export interface ItemsList {
+    type: string;
+    title: string;
+    items: object[];
+    buttonName: string;
+}
+
+async function getFavouriteSubscriptions() {
```

Comment:
For all fetch functions: i placed them in the lib to support reusage for our consumers / users of the lib

Reply:
Let's try to do this.